tests/test_admin_views.py

```diff
@@ -0,0 +1,129 @@
+import unittest
+
+from django.test import RequestFactory
+
+import pytest
+from OpenSSL import crypto
+
+from simple_certmanager.admin_views import (
+    CertificateSigningRequestInfoForm,
+    GenerateCertificateView,
+)
+from simple_certmanager.constants import CertificateTypes
+from simple_certmanager.models import Certificate
+
+
+class GenerateCertificateViewTestCase(unittest.TestCase):
```

Comment:
library tests should be in `pytest` style - we have a bunch of fixtures already in `tests/conftest.py` to generate private keys/certificates etc.